examples/pretrained_imagenet_scivision_example.ipynb

```diff
@@ -33,58 +55,33 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from scivision.io import load_pretrained_model\n",
-    "from scivision.catalog import catalog"
+    "default_catalog.models"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Load the database and query a model"
+    "Next, we identify datasources in the catalog that would be compatible with the model (based on `tasks`, `format` and `labels`/`labels_required`).  Let's use `data-003`, an image of a baby Koala."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "scrolled": true
+    "scrolled": false
    },
    "outputs": [],
    "source": [
-    "# Query a certain model\n",
-    "query_dict = {'model': 'https://github.com/quantumjot/scivision-test-plugin'}\n",
-    "\n",
-    "queryO = catalog.query(query_dict)\n",
-    "queryO"
+    "compatible_datasources = default_catalog.compatible_datasources('scivision-test-plugin')\n",
+    "compatible_datasources"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`query_dict`, in the above cell, can take other keys/values, e.g.:\n",
-    "\n",
-    "```python\n",
-    "query_dict = {'task': 'object-detection'}\n",
-    "```\n",
-    "\n",
-    "We can specify more than one key/value pair, and the specified conditioned will be joined by: `\" & \".join(queries)`\n",
-    "\n",
-    "---\n",
-    "\n",
-    "In the above cell, we queried two datasets stored here: \n",
-    "\n",
-    "https://github.com/alan-turing-institute/scivision/tree/main/scivision/catalog/data\n",
-    "\n",
-    "Also, you can access them via:\n",
-    "\n",
-    "```python\n",
-    "# models\n",
-    "catalog._catalog._models\n",
-    "# data sources\n",
-    "catalog._catalog._datasources\n",
-    "```"
+    "(To be replaced by a query-based interface on the catalog)"
```

Comment:
I just wanted to include a few built-in queries (like `compatible_models`) and leave this for another PR - it might benefit from a little discussion. The notebook says 'work in progress' at the top, so hope the comment isn't too distracting.